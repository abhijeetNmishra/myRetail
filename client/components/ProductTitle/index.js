import React from 'react';

const ProductTitle = (props) => {
  return (
    <div className="row h-text-center">
      <h2 className="title-product">
        <span>{props.title}</span>
        <span className="h-sr-only" aria-hidden="true">{props.title}</span>
      </h2>
    </div>
  );
};

React.propTypes = {
  title: React.PropTypes.string
};

export default ProductTitle;
