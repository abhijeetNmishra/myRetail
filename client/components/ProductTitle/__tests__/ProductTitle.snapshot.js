/* eslint-disable */
import React from 'react';
import renderer from 'react-test-renderer';
import ProductTitle from '../index';

test('ProductTitle renders correctly', () => {
  const title = 'test title';
  const productTitle = renderer.create(
    <ProductTitle title={title}></ProductTitle>
  ).toJSON();
  expect(productTitle).toMatchSnapshot();
});
