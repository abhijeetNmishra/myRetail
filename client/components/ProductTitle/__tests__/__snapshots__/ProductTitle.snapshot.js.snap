exports[`test ProductTitle renders correctly 1`] = `
<div
  className="row h-text-center">
  <h2
    className="title-product">
    <span>
      test title
    </span>
    <span
      aria-hidden="true"
      className="h-sr-only">
      test title
    </span>
  </h2>
</div>
`;
